securityinternal/v1/0000_04_securityinternal-openshift_01_rangeallocation.crd.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
make it v1 from the start?

Reply:
since it's internal only probably ok to make it v1 from the start

Reply:
> make it v1 from the start?

been around for six years.